featuretools/entityset/entityset.py

```diff
@@ -299,7 +298,8 @@ def add_column(self, entity_id, column_id, column_data, type=None):
         """
         Add variable to entity's dataframe
         """
-        self.entity_stores[entity_id].add_column(column_id, column_data, type=type)
+        self.entity_stores[entity_id].add_column(
```

Comment:
can you undo your line length changes for now? We currently allow lines to go beyond 80 characters in special cases like this one and edits below